sqlglot/expressions.py

```diff
@@ -937,6 +937,42 @@ def on(self, *expressions, append=True, dialect=None, copy=True, **opts):
 
         return join
 
+    def using(self, *expressions, append=True, dialect=None, copy=True, **opts):
```

Comment:
This isn't tested.

Reply:
Thanks for pointing out, I forgot to add more tests after the docstring.